base_phone/tests/test_base_phone.py

```diff
@@ -61,14 +61,14 @@ def test_partner_phone_formatting(self):
         # Create a partner with country
         parent_partner2 = rpo.create(
             {
-                "name": "C2C",
+                "name": "Parent Partner 2",
                 "country_id": self.env.ref("base.ch").id,
                 "is_company": True,
             }
         )
         partner2 = rpo.create(
             {
-                "name": "Joël Grand-Guillaume",
+                "name": "Partner 2",
                 "parent_id": parent_partner2.id,
                 "phone": "(0) 21 619 10 10",
                 "mobile": "(0) 79 606 42 42",
```

Comment:
These numbers are camptocamp ones. Please fully anonymise all the data.

Reply:
cc @nguyenminhchien

Reply:
Good day @StephaneMangin, new PR has been created to replace this PR and to update the test data.